src/pages/dmt5-trading-signals/index.tsx

```diff
@@ -13,47 +13,56 @@ import device from 'themes/device'
 import { usePageLoaded } from 'components/hooks/use-page-loaded'
 
 const meta_attributes = {
-    og_title: localize('Deriv MetaTrader 5 trading signals | Resources | Deriv'),
+    og_title: localize('_t_Deriv MetaTrader 5 trading signals | Resources | Deriv_t_'),
     og_description: localize(
-        'Subscribe to Deriv MetaTrader 5 trading signals to copy the trades of experienced traders, or become a signal provider and share your strategies.',
+        '_t_Subscribe to Deriv MetaTrader 5 trading signals to copy the trades of experienced traders, or become a signal provider and share your strategies._t_',
     ),
 }
 const signal_content_subscriber = {
```

Comment:
Instead of passing the element to these array let's pass this type to it:
```
type SignalProps = {
    content: {
        header: TString
        text: TString
        list: TString[]
    }
}
```

Reply:
```
export const Signal = ({ content }: SignalProps) => {
    return (
        <StyledSection background="var(--color-white)">
            <Flex direction="column" max_width="99.6rem" m="0 auto" jc="space-between" ai="center">
                <StyledText>
                    <Localize translate_text={content.text} />
                </StyledText>
                <ContentWrapper>
                    <StyledHeader>
                        <Localize translate_text={content.header} />
                    </StyledHeader>
                    <CardWrapper>
                        {content.list.map((text, idx) => {
                            return (
                                <Card key={idx}>
                                    <StyledChecklist />
                                    <CardText>{text}</CardText>
                                </Card>
                            )
                        })}
                    </CardWrapper>
                </ContentWrapper>
            </Flex>
        </StyledSection>
    )
}
```